tokio/src/runtime/runtime.rs

```diff
@@ -421,6 +421,7 @@ impl Runtime {
     /// # Examples
     ///
     /// ```
+    /// # if cfg!(miri) { return } // Miri reports error when main thread terminated without waiting all remaining threads.
```

Comment:
Specifically we report a leak; this can be ignored -- but maybe you want to detect memory leaks in the other tests.

Reply:
We definitely want the leak detection for lots of other tests.